code/modules/vehicles/mecha/working/ripley.dm

```diff
@@ -138,6 +138,21 @@
 	var/obj/item/mecha_parts/mecha_equipment/mining_scanner/scanner = new
 	scanner.attach(src)
 
+/obj/vehicle/sealed/mecha/working/ripley/cargo
+	desc = "An ailing, old, repurposed cargo hauler"
+	name = "\improper APLU \"Cargo\""
```

Comment:
this name sucks ass just keep it default of think of something better

Reply:
Could just name it "Power Loader", while also removing "APLU" since that just means "Autonomous Power Loader Unit". "Hauler" works if you are feeling uncreative. Ultimately, the default name with something like "old", "early" or "prototype" prefixed would work too.

Reply:
better name coming right up